packages/ui-module-header/src/components/profile.tsx

```diff
@@ -0,0 +1,47 @@
+import Image from 'next/image';
+import { ProfileProps } from '../types';
+import { Dropdown, Icon } from '@test-assessment/ui-components';
+import React from 'react';
+
+export const Profile = (props: ProfileProps) => {
+  const { name, image } = props;
+  const [open, setOpen] = React.useState(false);
+  const styles = useStyles();
+
+  return (
+    <Dropdown
+      button={
+        <div className={`${styles.root}`}>
+          <Image src={image} alt={''} />
+          <div className={`${styles.name}`}>{name}</div>
```

Comment:
Could you add the truncate class to hide large text ?  `truncate overflow-hidden`:
https://tailwindcss.com/docs/overflow#hiding-content-that-overflows